BlackJackets-ui/src/pages/VenueAdd.jsx

```diff
@@ -98,6 +100,15 @@ export default function VenuePage() {
                   required
                 />
               </div>
+               <div className='form-group mb-2'>
+                  <label className='form-label'>Video</label>
+                    <input
+                     type='text'
+                     name='video'
+                     value={video}
+                     className='form-control'
+                     onChange={(e) => setVideo(e.target.value)} required />
```

Comment:
I think `required` could be removed in the case that a Venue doesn't have a video to upload when they register